Makefile

```diff
@@ -391,7 +391,8 @@ ensure-codespell:
 # Dependencies:
 #   - ensure-dprint: Ensures that dprint is installed and available in the system.
 lint-toml: ensure-dprint
-	dprint fmt
+	cargo metadata --format-version=1 --no-deps | jq '.packages.[].manifest_path' | \
+	xargs dprint fmt --config dprint.json
```

Comment:
just dprint fmt is a lot simpler than this

Reply:
frequently `dprint fmt` doesn't work for me from crate root. need to explicitly `cd` into the specific crate giving problems in ci and run `dprint fmt`

Reply:
and from reading `dprint --help` that made sense
```
GETTING STARTED:
  1. Navigate to the root directory of a code repository.
  2. Run `dprint init` to create a dprint.json file in that directory.
  3. Modify configuration file if necessary.
  4. Run `dprint fmt` or `dprint check`.
```

Reply:
same for clippy from workspace root sometimes, doesn't catch everything that it catches if you manually navigate into the specific crate. clippy doesn't cause so much trouble in ci though.